pom.xml

```diff
@@ -310,6 +311,26 @@
                 <artifactId>mockito-core</artifactId>
                 <version>${mockito.version}</version>
                 <scope>test</scope>
+                <exclusions>
```

Comment:
I'd prefer adding Objensis to dependency management instead of exclusions:

```
   <dependency>
                <groupId>org.objenesis</groupId>
                <artifactId>objenesis</artifactId>
                <version>${objenesis.version}</version>
            </dependency>
```